src/App.tsx

```diff
@@ -1,24 +1,221 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import {
+  addTodo,
+  getTodos,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+
+const USER_ID = 7011;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [title, setTitle] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState<Status>(Status.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isDisabledInput, setIsDisabledInput] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [deletingTodoId, setDeletingTodoId] = useState(0);
```

Comment:
change it to `loadingTodoIds` to:
- show loader for todo that are updating (status/title change)
- `Ids` in the plural form - save an array instead of one id (empty by default) to show loader for todos during multiple change actions (clear completed/toggle all)